examples/lab-robust/v2/qwiklabs.yaml

```diff
@@ -78,64 +62,34 @@ environment:
         allow_vpc_deletion: false
         allowed_rds_instances: ['db.t2.micro']
   student_visible_outputs:
-    - label:
-        locales:
-          en: "Open GCP Console"
+    - label: Open GCP Console
       reference: primary_project.console_url
-    - label:
-        locales:
-          en: "GCP Project"
+    - label: GCP Project
       reference: primary_project.project_id
-    - label:
-        locales:
-          en: "GCP Username"
+    - label: GCP Username
       reference: primary_user.username
-    - label:
-        locales:
-          en: "GCP Password"
+    - label: GCP Password
       reference: primary_user.password
-    - label:
-        locales:
-          en: "InstanceDns"
+    - label: InstanceDns
       reference: primary_project.startup_script.InstanceDns
-    - label:
-        locales:
-          en: 'AWS Account Number'
+    - label: AWS Account Number
       reference: the_account.account_number
-    - label:
-        locales:
-          en: 'Username'
+    - label: Username
       reference: the_account.username
-    - label:
-        locales:
-          en: 'password'
+    - label: password
       reference: the_account.password
-    - label:
-        locales:
-          en: 'AWS Console URL'
+    - label: AWS Console URL
       reference: the_account.console_url
 
 assessment:
   passing_percentage: 75.0
   steps:
-  - title:
-      locales:
-        en: Create a Cloud Storage bucket
-        es: Crear un depósito de almacenamiento en la nube
+  - title: Create a Cloud Storage bucket
     maximum_score: 5
     student_messages:
-      success:
-        locales:
-          en: Great job! You created the bucket!
-          es: ¡Gran trabajo! ¡Creaste el cubo!
-      bucket_missing:
-        locales:
-          en: Oops! No bucket found.
-          es: ¡Uy! No se ha encontrado el cubo.
-      bucket_misconfigured:
-        locales:
-          en: Hmm. The bucket is there, but it is misconfigured.
-          es: Hmm. El cubo está allí, pero está mal configurado.
+      success: Great job! You created the bucket!
+      bucket_missing: Oops! No bucket found.
+      bucket_misconfigured: Hmm. The bucket is there, but it is misconfigured.
     services:
     - primary_project.StorageV1
     code: |-
```

Comment:
Best practice is going to be to have the code in separate files in the `assessments` dir and named `step_one_check.rb` with matching `method_name:`